### ISSUE TITLE
camera.get_model_matrix Unable to convert function return value to a Python type

### ISSUE THREAD

**Issue by tairch** [intel-isl/Open3D#3117]
**Describe the bug**
camera.get_model_matrix() returnes 

```
Traceback (most recent call last):
  File "open3d_2.py", line 165, in <module>
    modelMatrix = camera.get_model_matrix()
TypeError: Unable to convert function return value to a Python type! The signature was
        (self: open3d.cpu.pybind.visualization.rendering.Camera) -> Eigen::Transform<float,3,2,0>
```

the same problem exists with camera.get_projection_matrix() and camera.get_view_matrix().

**To Reproduce**
```
gui.Application.instance.initialize()
w = o3d.visualization.O3DVisualizer("03DVisualizer",640, 480)
gui.Application.instance.add_window(w)
open3DScene = w.scene
camera = open3DScene.camera
modelMatrix = camera.get_model_matrix() //or camera.get_projection_matrix() or camera.get_view_matrix()
```
**Expected behavior**
getting the model matrix, the projection matrix and the view matrix

**Environment (please complete the following information):**

- Operating system: Windows 10 64-bit
- Python version: Python 3.8
- Open3D version: 0.12.0+711742a (latest right now)
- Is this remote workstation?: no
- How did you install Open3D?: pip with wheel from the getting started page on the docs

**Comment by prewettg** [comment]
The above fix should be in `master` within a few days and fixed in 0.13.

Just fyi, you'll find that the above script crashes, though, due to Filament not getting cleaned up. Right now we assume that you're going to call `gui.Application.run()`, which cleans up when it is done. Also, you'll probably want to call `w.reset_camera_to_default()` to get a more usable view when you add geometries.

**Comment by tairch** [comment]
Perfect, thank you!
